lib/grape-swagger/doc_methods.rb

```diff
@@ -47,6 +47,12 @@ def parse_params(params, path, method)
                           'double'
                         when 'Symbol'
                           'string'
+                        when /^\[(.*)\]$/
```

Comment:
Maybe use a named field here, like type. That avoids having a `last_match[1]` and is probably clearer.